frontend/scripts/react-components/nav/filters-nav/recolor-by-selector/recolor-by-selector.component.jsx

```diff
@@ -1,17 +1,18 @@
 /* eslint-disable jsx-a11y/no-static-element-interactions,jsx-a11y/no-noninteractive-element-interactions */
 import React, { Component } from 'react';
-import classNames from 'classnames';
-import _ from 'lodash';
-import Tooltip from 'react-components/tool/help-tooltip.component';
-import Dropdown from 'react-components/tool/nav/dropdown.component';
-import RecolorByNodeLegendSummary from 'containers/tool/nav/recolor-by-node-legend-summary.container';
 import PropTypes from 'prop-types';
+import Tooltip from 'react-components/tool/help-tooltip.component';
+import FiltersDropdown from 'react-components/nav/filters-nav/filters-dropdown.component';
+import RecolorByNodeLegendSummary from 'react-components/nav/filters-nav/recolor-by-selector/recolor-by-node-legend-summary/recolor-by-node-legend-summary.container';
+import cx from 'classnames';
```

Comment:
happy to rename this if you don't like "classnames" but cx is not informative. Let's improve this

Reply:
`cx` is a convention, and it's actually used in their docs.